### PR TITLE
Grid component

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Please ignore modal component

image

image

### CODE REVIEW

File: styles/06-components/modal.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+/* }
+
+.modal {
+  width: clamp(10rem, 20rem, 30rem);;
+  
+}
+
+.modal .icon {
+  color: var(--color-secondary)
+} */
```
Comment (xii-vi) [inline styles/06-components/modal.css:25]: Remove the comments if it is of no use.

File: styles/06-components/brand-theme.css [hunk 2]
```diff
@@ -17,4 +17,9 @@ body {
   min-height: 400px;
```
Comment (xii-vi) [inline styles/06-components/brand-theme.css]: try to use rem instead of px

File: styles/06-components/modal.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ 
+
+
+.modal {
+  position: fixed;
+  right: 0;
+  border: 2px solid var(--color-secondary); 
+  height: 350px;
```
Comment (xii-vi) [inline styles/06-components/modal.css]: try to use rem instead of px

**Review by xii-vi — APPROVED** [review]
Please, go through the comments before merging. Also includes visual components for better understanding.

File: styles/06-components/modal.css (hunk 1 above)
Comment (HarshnaNHaswani) [reply styles/06-components/modal.css:25]: I'll need it to make it responsive later

File: styles/06-components/brand-theme.css (hunk 2 above)
Comment (HarshnaNHaswani) [reply styles/06-components/brand-theme.css]: Will do! Thanks for pointing it out

File: styles/06-components/modal.css (hunk 3 above)
Comment (HarshnaNHaswani) [reply styles/06-components/modal.css]: Will make the necessary changes

**Comment by HarshnaNHaswani** [comment]
Thanks for your time and the review @xii-vi , will do the necessary changes

**Review by sahiba0915 — COMMENTED** [review]
overall everything is fine just change the absolute units to relative units wherever possible

**Comment by HarshnaNHaswani** [comment]
> overall everything is fine just change the absolute units to relative units wherever possible

Changed! Thanks for your time and the review @sahiba0915

**Pull request merged**